File: Examples/VisualBasic/Loading-and-Saving/CreateDocument.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Words

Public Class CreateDocument
    Public Shared Sub Run()
        ' ExStart:CreateDocument            
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_LoadingAndSaving()

        ' Initialize a Document.
        Dim doc As New Document()

        ' Use a document builder to add content to the document.
        Dim builder As New DocumentBuilder(doc)
        builder.Writeln("Hello World!")

        dataDir = dataDir & Convert.ToString("CreateDocument_out_.docx")
        ' Save the document to disk.
        doc.Save(dataDir)

        ' ExEnd:CreateDocument

        Console.WriteLine(Convert.ToString(vbLf & "Document created successfully." & vbLf & "File saved at ") & dataDir)

    End Sub
End Class

File: Examples/CSharp/Loading-and-Saving/CreateDocument.cs
using System;
using System.Collections.Generic;
using Aspose.Words;

namespace Aspose.Words.Examples.CSharp.Loading_Saving
{
    class CreateDocument
    {
        public static void Run()
        {
            //ExStart:CreateDocument            
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_LoadingAndSaving();

            // Initialize a Document.
            Document doc = new Document();
            
            // Use a document builder to add content to the document.
            DocumentBuilder builder = new DocumentBuilder(doc);
            builder.Writeln("Hello World!");  
        
            dataDir  = dataDir + "CreateDocument_out_.docx";
            // Save the document to disk.
            doc.Save(dataDir);
           
            //ExEnd:CreateDocument

            Console.WriteLine("\nDocument created successfully.\nFile saved at " + dataDir);

        }
    }
}
